### PR TITLE
Updates for Ruby 2.7

### PR DESCRIPTION
**Pull request by rchekaluk**
This pull request primarily addresses a warning that appears with Ruby 2.7 (sources [here](https://bloggie.io/@kinopyo/how-to-fix-ruby-2-7-warning-using-the-last-argument-as-keyword-parameters-is-deprecated) and [here](https://piechowski.io/post/last-arg-keyword-deprecated-ruby-2-7/)):
```
Using the last argument as keyword parameters is deprecated
```
It also addresses this warning:
```
Use assert_nil if expecting nil
```

### CODE REVIEW

File: lib/tax_cloud/errors/tax_cloud_error.rb [hunk 1]
```diff
@@ -44,7 +44,7 @@ def compose_message(key, attributes = {})
       #
       # Returns a localized error message string.
       def translate(key, options)
-        ::I18n.translate("#{BASE_KEY}.#{key}", { locale: :en }.merge(options)).strip
+        ::I18n.translate("#{BASE_KEY}.#{key}", **({ locale: :en }.merge(options))).strip
```
Comment (drewtempelmeyer) [inline lib/tax_cloud/errors/tax_cloud_error.rb:47]:
```suggestion
        ::I18n.translate("#{BASE_KEY}.#{key}", **{ locale: :en }.merge(options)).strip
```

**Comment by drewtempelmeyer** [comment]
@rchekaluk thanks for updating this. Can you also add 3.0.0 and 3.1.0 to the .travis.yml config?

**Comment by rchekaluk** [comment]
Done

**Comment by rchekaluk** [comment]
@drewtempelmeyer Ping

**Comment by dblock** [comment]
Looks like Travis is now permanently MIA. Can someone (you?) PR a replacement with GHA before we merge any new code?

**Comment by dblock** [comment]
https://github.com/txcrb/tax_cloud/issues/53

**Comment by rchekaluk** [comment]
Done https://github.com/txcrb/tax_cloud/pull/54.

**Comment by dblock** [comment]
Want to resurrect this on top of #54?

**Comment by rchekaluk** [comment]
Done

**Pull request merged**